sklearn/cluster/hierarchical.py

```diff
@@ -74,18 +73,31 @@ def _fix_connectivity(X, connectivity, affinity):
                 connectivity[idx_i[ii], idx_j[jj]] = True
                 connectivity[idx_j[jj], idx_i[ii]] = True
 
-    return connectivity, n_components
+    return connectivity, n_connected_components
 
 
 def _single_linkage_tree(connectivity, n_samples, n_nodes, n_clusters,
-                         n_components, return_distance):
+                         n_components, n_connected_components,
```

Comment:
I don't think the user ever passes this parameter, so I don't think anything needs changing here but the name. No need to deprecate

Reply:
Thanks understood, I updated that in commit `ff8b930`